### ISSUE TITLE
K9 crashes when opening any email while Chrome is disabled

### ISSUE THREAD

**Issue by jcarius** [k9mail/k-9#3821]
### Expected behavior
The (HTML) email should be displayed with android web view or a fallback solution (or at least report that an HTML renderer was not found).

### Actual behavior
K9 crashes: "App stopped working"

### Steps to reproduce
1. Disable the Chrome Browser app
2. Try to open any email

### Environment
K-9 Mail version: 5.6

Android version: 8.0.0

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

**Comment by ByteHamster** [comment]
What is selected in your Android settings `Developer Options` -> `WebView implementation`? K9 should just automatically use the one that is selected there.

**Comment by jcarius** [comment]
yes, usually there is Chrome selected, but when I disable it it switches automatically to "Android System WebView 66.0.3359.126"
